generator/sbpg/targets/resources/sbp_base.tex

```diff
@@ -32,6 +32,7 @@
 \numberwithin{table}{subsection}
 \numberwithin{field}{subsection}
 
+\renewcommand{\version}{(((version)))}
```

Comment:
I can't inside the the .sty file, so in order to do this I'd have to move the footnote stuff out of the .sty file